.gitignore

```diff
@@ -20,4 +20,4 @@ __debug_bin
 
 main
 dist/
-
+cmd/zinc
```

Comment:
What exactly would this achieve? Would this not lead to ignoring main.go ?

Reply:
the binary zinc should not commit to git, so ignore it.

```
cd zinc/cmd
go build
```
will generate binary zinc, so i ignore: `cmd/zinc`